modules/genomic_browser/tools/HumanMethylation450k_annotations_to_sql.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
```

Comment:
Python dependency : This should be noted in the readme for this module - if you want to use/do X, install python to run Y script
(future ticket: rewrite in PHP?)